examples/flutter_gallery/lib/demo/material/snack_bar_demo.dart

```diff
@@ -83,6 +83,11 @@ class _SnackBarDemoState extends State<SnackBarDemo> {
         // can refer to the Scaffold with Scaffold.of().
         builder: buildBody
       ),
+      floatingActionButton: FloatingActionButton(
+        child: const Icon(Icons.add),
+        tooltip: 'Create',
+        onPressed: () {}
```

Comment:
When we have dummy calls to action, put in a print to console statement onPressed that's like "Floating Action Button was pressed" so something happens.

Reply:
Done